spec/unit/target/build_settings/aggregate_target_settings_spec.rb

```diff
@@ -465,6 +465,48 @@ def pod_target(spec, target_definition)
             @generator.other_ldflags.should == %w(-ObjC -l"StaticLibrary" -l"VendoredDyld" -l"xml2" -framework "PodTarget" -framework "VendoredFramework" -framework "XCTest")
```

Comment:
@segiddins I don't understand why `-l"xml2"` or `-framework "XCTest"` are included in the settings of this aggregate target since we are using frameworks...

Reply:
wait I think I do...we always propagate these to the aggregate target even if we use frameworks...